packages/scripts/README.md

```diff
@@ -104,20 +88,21 @@ This script automatically use the optimized config but sometimes you may want to
 
 -   `--webpack-bundle-analyzer` – enables visualization for the size of webpack output files with an interactive zoomable treemap.
 -   `--webpack-copy-php` – enables copying all PHP files from the source directory ( default is `src` ) and its subfolders to the output directory.
--   `--webpack-no-externals` – disables scripts' assets generation, and omits the list of default externals.
--   `--webpack-src-dir` – Allows customization of the source code directory. Default is `src`.
+-   `--webpack-no-externals` – disables scripts’ assets generation, and omits the list of default externals.
+-   `--source-path` – Allows customization of the source directory. Default is `.`. In the automatic detection mode the default changes to `src`.
```

Comment:
What does "automatic detection mode" mean? I see there's another section on `### Automatic block.json detection and the source code directory` - it would be good to link to that. That can be in a follow-up so we know what the link is.